cmd/kubeadm/app/phases/upgrade/versiongetter.go

```diff
@@ -135,9 +134,6 @@ func NewOfflineVersionGetter(versionGetter VersionGetter, version string) Versio
 func (o *OfflineVersionGetter) VersionFromCILabel(ciVersionLabel, description string) (string, *versionutil.Version, error) {
 	if o.version == "" {
 		versionStr, version, err := o.VersionGetter.VersionFromCILabel(ciVersionLabel, description)
-		if err == nil {
-			fmt.Printf("[upgrade/versions] Latest %s: %s\n", description, versionStr)
-		}
 		return versionStr, version, err
```

Comment:
you can `return o.VersionGetter.VersionFromCILabel(ciVersionLabel, description)` directly here

please keep the commits squashed to one.